integ-test/src/yamlRestTest/resources/rest-api-spec/test/issues/3102.yml

```diff
@@ -0,0 +1,53 @@
+setup:
+  - skip:
+      features:
+        - headers
+  - do:
+      indices.create:
+        index: test
+        body:
+          settings:
+            max_result_window: 1
+  - do:
+      bulk:
+        index: test
+        refresh: true
+        body:
+          - '{"index": {}}'
+          - '{"id": 1}'
+          - '{"index": {}}'
+          - '{"id": 2}'
+          - '{"index": {}}'
+          - '{"id": 3}'
+
+
+---
+teardown:
+  - do:
+      query.settings:
+        body:
+          transient:
+            plugins.calcite.enabled : false
+            plugins.calcite.fallback.allowed : true
```

Comment:
why we need this teardown step? it seems not enable calcite in setup step. 

@penghuo is each xyx.yml independent?